app/src/ui/clone-repository/clone-github-repository.tsx

```diff
@@ -62,6 +63,9 @@ interface ICloneGithubRepositoryProps {
    * available for cloning.
    */
   readonly onRefreshRepositories: (account: Account) => void
+
+  /** Initiate cloning if a repository and cloning path are selected and no current errors. */
+  readonly cloneIfCloningEnabled: () => void
```

Comment:
**Non-blocking**
I think a name like `onRowKeyDown` does a better job describing what the purpose of the prop is. We're also using that name in other components.